src/SignalR/clients/cpp/include/signalrclient/connection.h

```diff
@@ -29,16 +28,16 @@ namespace signalr
 
         connection& operator=(const connection&) = delete;
 
-        SIGNALRCLIENT_API pplx::task<void> __cdecl start();
+        SIGNALRCLIENT_API void __cdecl start(std::function<void(std::exception_ptr)> callback) noexcept;
```

Comment:
Can these callbacks be marked noexcept too? `std::function<void(std::exception_ptr) noexcept>`

Reply:
I thought about it, but I don't think it works
https://godbolt.org/z/dI2GuJ